packages/redis_client/lib/src/redis_client.dart

```diff
@@ -73,153 +141,169 @@ class RedisClient {
   /// A completer which completes when the client establishes a connection.
   var _connected = Completer<void>();
 
+  /// Whether the client is connected.
+  var _isConnected = false;
+
   /// A completer which completes when the client disconnects.
   /// Begins in a completed state since the client is initially disconnected.
   var _disconnected = Completer<void>()..complete();
```

Comment:
I thought member variables were typically declared with types in Dart?

Reply:
The lint rules we have consistently enforce omitting types if they can be inferred.